docs/podman-inspect.1.md

```diff
@@ -80,25 +86,57 @@ Display the total file size if the type is a container
     "Annotations": {}
 }
 ```
-
+Inspect a container by its shortened ID and only output its *ImageName*.
 ```
 # podman inspect a04 --format "{{.ImageName}}"
 fedora
 ```
 
+Display only the *capabilities* of the latest container.
 ```
-# podman inspect a04 --format "{{.GraphDriver.Name}}"
-overlay
+podman inspect --latest --format {{.EffectiveCaps}}
+[CAP_CHOWN CAP_DAC_OVERRIDE CAP_FSETID CAP_FOWNER CAP_MKNOD CAP_NET_RAW CAP_SETGID CAP_SETUID CAP_SETFCAP CAP_SETPCAP CAP_NET_BIND_SERVICE CAP_SYS_CHROOT CAP_KILL CAP_AUDIT_WRITE]
 ```
 
+Generate kubernetes YAML describing a container by ID.
 ```
-# podman inspect --format "size: {{.Size}}" alpine
-size:   4405240
-```
+apiVersion: v1
```

Comment:
seems to be missing the actual podman command to make this output.

Reply:
nice catch